include/alpaka/mem/buf/Traits.hpp

```diff
@@ -29,6 +29,12 @@ namespace alpaka
         template<typename TElem, typename TDim, typename TIdx, typename TDev, typename TSfinae = void>
         struct AsyncBufAlloc;
 
+        //! The stream-ordered memory allocator capability trait.
+        template<typename TDim, typename TDev>
+        struct HasAsyncBufferSupport : public std::false_type
```

Comment:
Why not

```suggestion
        struct HasAsyncBufSupport : public std::false_type
```

? This would keep the trait consistent with the rest of the `Buf` API.

Reply:
Sure.